### PR TITLE
Getting the DOMNode from the refs to pass to Highstock.

### PR DESCRIPTION
**Pull request by JohnPhoto**
The previous code was not working with React 0.14.3

### CODE REVIEW

**Comment by kirjs** [comment]
Thanks a lot for the PR, can you please take a look why the tests got broken?
Also a link to description of the change in React which causes this fix would be helpful.

**Comment by JohnPhoto** [comment]
I think I actually found the reason to why it did not work for me the first time, I will try around some more, but it seems like the Highcharts works a bit differently now so the "require" with globals and such have changed a bit. Will update PR when ready.

**Comment by JohnPhoto** [comment]
The problem was basically that the new highcharts did not only wanted to load the HC-adapter from global but you actually have to inject the adapter into highcharts for it to work properly. And then expose the result on the global context.

**Comment by JohnPhoto** [comment]
O btw, as noted in one of the commits, I did not manage to get the highcharts-more example to work, and I don't really know why. Atleast the tests are working.

**Comment by kirjs** [comment]
I didn't realize the build was broken because of Highcharts-release update, and not something you did, so thanks a lot for the research. I am going to take a look at more.js some time next week, and merge it in.

**Comment by JohnPhoto** [comment]
I found out how to the get the HC-More stuff worked.

File: test/bundleSpec.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ require('./simulateDOM');
 
 
 function bundleTest(bundleName, chartName) {
-  describe('react-highcharts/bundle' + bundleName, function () {
+  describe('react-highcharts/' + bundleName, function () {
```
Comment (kirjs) [inline test/bundleSpec.js]: Why is the bundle part gone here?

Comment (JohnPhoto) [inline test/bundleSpec.js]: Because "bundleName" already contains "bundle".

Comment (kirjs) [inline test/bundleSpec.js]: Got it

**Comment by kirjs** [comment]
Would it be backwards compatible with react 0.14.1?

**Comment by JohnPhoto** [comment]
I do not know, know of any breaking changes that you have conformed with?
I have not tried any other version than defined here.

**Comment by kirjs** [comment]
Hm, so I tried it earlier this week, and the demo worked well, now I'm getting an error saying Highcharts is not a function.

**Comment by kirjs** [comment]
OK, I'll merge it in, and then will figure out the rest

**Comment by kirjs** [comment]
Thanks a lot for your work!

**Pull request merged**